Integrations/Autodesk/maya/scripts/unityCommands.mel

```diff
@@ -68,7 +68,7 @@ proc storeAttribute(string $node, string $attr, string $attrValue){
 
 
 proc int setExists(string $setName){
-    return stringArrayContains($setName, `listSets -allSets`);
+    return stringArrayContains($setName, `ls -sets`);
```

Comment:
listSets -allSets was not giving the full name of the set (including namespaces)

Reply:
Classic MEL